package/libstorage.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon May 18 15:01:44 UTC 2015 - mvidner@suse.com
+
+- Added stub error strings. Now you should see
+  "DISK_PARTITION_NOT_FOUND" instead of "-1008".
```

Comment:
Actually in addition to the number, but that depends on the user of the library.